projects/storefrontapp/src/app/app.module.ts

```diff
@@ -65,7 +65,7 @@ if (!environment.production) {
         fallbackLang: 'en',
       },
       features: {
-        level: '1.2',
+        level: '1.3',
```

Comment:
do we keep this? or keep it for 1.2?

Reply:
Talked to Gil, we will leave it, so that it is enabled on all the servers and in our local development.